_pages/en_US/faq.md

```diff
@@ -6,30 +6,42 @@ redirect_from:
+### How can I dump my cartridges?
+To dump your cartridges for use with emulators, nds-bootstrap or flashcarts, follow [this guide](dumping-cartridges).
+
+### How do I play Nintendo DS cartridge dumps?
+Playing cartridge dumps on the console requires the use of a flashcart or nds-bootstrap, a program which enables games to be played from the internal SD card by redirecting slot-1 reads and writes to it.
+- With TWiLight Menu++ you can navigate your SD card to find ROM files to play with nds-bootstrap. The advantages to using TWiLight Menu++ are having a cheat menu, per-game settings, and avoiding the restrictions that forwarders bring. In other words, you can drop your ROM files directly and play without any setup. There is no 39 title limit, neither hiyaCFW or Unlaunch are required and there are no restrictions on SD card free space you can have.
+- hiyaCFW users can create [forwarders](nds-bootstrap-forwarders) for the SDNAND's DSi Menu, but it has some limitations. There is a hard limit of 39 titles, they are less convenient to make and they do not automatically patch Anti-Piracy.
+
+### How can I restore my NAND backup if I cannot boot the console?
+You can follow Gadorach's [hardmodding guide](https://web.archive.org/web/20151102221503/https://gbatemp.net/threads/dsi-downgrading-the-complete-guide.393682/) to hardmod your DSi. Previous soldering experience is required.
 
 ### How do I update my Nintendo DSi Homebrew?
-- **Unlaunch** - Download `UNLAUNCH.DSI` and run it through the Unlaunch menu
-  - Hold (A) & (B) when turning on your console
+- **Unlaunch** - Follow the instructions on the [Installing Unlaunch](/installing-unlaunch) page
 - **hiyaCFW** - Replace `hiya.dsi` on the root of the SD card
 - **TWiLight Menu++** - Follow Section 1b of [Launching the Exploit](launching-the-exploit#twilight-menu)
-  - If you are moving from DSiMenu++ or SRLoader, rename `sd:/_nds/SRLoader` or `sd:/_nds/DSiMenuPlusPlus` to `sd:/_nds/TWiLightMenu` & delete `53524C41` (and `534C5254`, if exists) at `sd:/title/00030015`
-  - Starting with v6.8.3, DS game saves are stored in a `saves` folder in the exact same location as the DS (`.nds`) ROMs. Please move them to that folder before updating
-  - Starting with v12.0.0, please convert any photos for the DSi theme to `.png`
+   - If you are moving from DSiMenu++ or SRLoader, rename `sd:/_nds/SRLoader` or `sd:/_nds/DSiMenuPlusPlus` to `sd:/_nds/TWiLightMenu` & delete `53524C41` (and `534C5254`, if exists) at `sd:/title/00030015`
+   - Starting with v6.8.3, DS game saves are stored in a `saves` folder in the exact same location as the DS (`.nds`) ROMs. Please move them to that folder before updating
+   - Starting with v12.0.0, please convert any photos for the DSi theme to `.png`
+   - The Wood UI (aka Acekard) theme has been temporarily removed, due to bugginess.
```

Comment:
No real need to copy this here when its got its own wiki
```suggestion
- **TWiLight Menu++** - Follow the [How to update TWiLight Menu++](https://github.com/DS-Homebrew/TWiLightMenu/wiki/How-to-update-TWiLight-Menu) instructions in its wiki
```

Reply:
I'll have to commit this separately